tools/data_processing/fix_Noflares.py

```diff
@@ -0,0 +1,38 @@
+#!usr/env/bin python
+
```

Comment:
This looks like a lot of copy/paste from the other files and should likely be consolidated.